src/boxed.tsx

```diff
@@ -13,16 +13,17 @@ type StylesProps = {
 
 const useStyles = createUseStyles(({colors, isDarkMode}) => ({
     boxed: {
-        background: ({isInverseInside}: StylesProps) =>
+        backgroundColor: ({isInverseInside}: StylesProps) =>
             isInverseInside && !isDarkMode ? colors.backgroundBrand : colors.backgroundContainer,
+        backgroundClip: ({isInverseInside}: StylesProps) => (isInverseInside ? 'padding-box' : 'border-box'), // for transparent borders, no background is drawn beneath the border.
```

Comment:
I thought these changes were to avoid this bottom border here:
![image](https://user-images.githubusercontent.com/1849576/134873647-6f807fc8-9942-4179-a3bb-e0bf621757e4.png)

Is it working fine?

Reply:
Well, in this case is only that the choosen image has a bottom pixel with transparency:
![image](https://user-images.githubusercontent.com/681159/134881115-1bed13cc-a715-4549-83f3-e13926a611e0.png)
without this fix, it was worse :stuck_out_tongue:  https://bit.ly/3ALqlIm

Reply:
can you find a better image for the screenshot?

Reply:
done